L/libsingular_julia/common.jl

```diff
@@ -5,11 +5,11 @@ import Pkg: PackageSpec
 import Pkg.Types: VersionSpec
 
 name = "libsingular_julia"
-version = VersionNumber(0, 9, 200 + julia_version.minor)
+version = VersionNumber(0, 10, julia_version.minor)
```

Comment:
Is removing the 200 correct?  Sorry, I don't follow the versioning scheme of the package :slightly_smiling_face:

Reply:
To simuate patch number this was introduced here: https://github.com/fingolfin/Yggdrasil/commit/850197a6ec28b4c28ddc0ba06fb621cdbf84db30#diff-92b56ff034ba8b37e5e99a14e623b39088eaad34e0f223c963c24b885f3c3252

Reply:
we will go with 0.10.0